### PR TITLE
Add support for oneaccess router

### PR DESCRIPTION
**Pull request by mathieu-artic**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by mathieu-artic** [comment]
Hi !
I don't understand why librenms don't use the OID  for the CPU description
image
Do you help me ?
Thx !

**Comment by Jellyfrog** [comment]
Congratulations on your first PR!

Would you mind removing the `.mib` file extension from the MIBs?

File: mibs/oneaccess/ONEACCESS-SYS-MIB.mib [hunk 1]
```diff
@@ -0,0 +1,483 @@
+--          *****************************
```
Comment (PipoCanaja) [inline mibs/oneaccess/ONEACCESS-SYS-MIB.mib:1]: Hi
Could you rename the mib file without the ".mib" extension ?
Thx

File: mibs/oneaccess/ONEACCESS-GLOBAL-REG.mib [hunk 2]
```diff
@@ -0,0 +1,263 @@
+--          *****************************
```
Comment (PipoCanaja) [inline mibs/oneaccess/ONEACCESS-GLOBAL-REG.mib:1]: Same here, could you remove the .mib extension?

File: mibs/oneaccess/ONEACCESS-SYS-MIB.mib (hunk 1 above)
Comment (mathieu-artic) [reply mibs/oneaccess/ONEACCESS-SYS-MIB.mib]: Hi ! 
it's good now

File: includes/definitions/discovery/oneaccess.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        version: ONEACCESS-SYS-MIB::oacSysIMSysMainBSPVersion.0
+        serial: ONEACCESS-SYS-MIB::oacExpIMSysHwcSerialNumber.0
+        hardware: ONEACCESS-SYS-MIB::oacExpIMSysHwcProductName.0
+
+    processors:
+        data:
+            -
+                oid: oacSysCpuUsedValue
+                num_oid: '.1.3.6.1.4.1.13191.10.3.3.1.2.3.1.3.{{ $index }}'
+                descr: ONEACCESS-SYS-MIB::oacSysIMSysMainCPU
```
[the remark this answers is not in the document]
Comment (mathieu-artic) [reply includes/definitions/discovery/oneaccess.yaml]: Thank's ! It's work great
image

I need to update tests files ?

[the remark this answers is not in the document]
Comment (mathieu-artic) [reply includes/definitions/discovery/oneaccess.yaml]: tests files updated

[the remark this answers is not in the document]
Comment (mathieu-artic) [reply includes/definitions/discovery/oneaccess.yaml]: Oh, it's text; on includes/definitions/oneaccess.yaml, fixed ...

File: tests/data/oneaccess_lbb350.json [hunk 4]
```diff
@@ -0,0 +1,2598 @@
...
+        "discovery": {
+            "processors": [
+                {
+                    "entPhysicalIndex": 0,
+                    "hrDeviceIndex": 0,
+                    "processor_oid": ".1.3.6.1.4.1.13191.10.3.3.1.2.3.1.3.0",
+                    "processor_index": "0",
+                    "processor_type": "oneaccess",
+                    "processor_usage": 71,
+                    "processor_descr": "Freescale T1040E (1.1) - Security Engine - 8-port Ethernet switc",
```
Comment (PipoCanaja) [inline tests/data/oneaccess_lbb350.json:2397]: Not really sure about the Processor name here. Seems that 4 CPUs are discovered. 1st one with this "Freescale ..." name, and the 3 others with just "Processor". If it is correct for the 1st one, may be you could find a way to have something a little bit more precise for the 3 others (so at least we can differentiate them). 

That being said, it looks very good.

Comment (mathieu-artic) [reply tests/data/oneaccess_lbb350.json:2397]: This Freescale is a 4 Core CPU
image
https://www.nxp.com/docs/en/fact-sheet/T1FAMILYFS.pdf
OneAccess use only one OID for the name of the CPU ...
"Description of the main CPU used on the main board"

Comment (mathieu-artic) [reply tests/data/oneaccess_lbb350.json:2397]: @PipoCanaja Hi, it's ok or not ?

Comment (murrant) [reply tests/data/oneaccess_lbb350.json:2397]: @mathieu-oui I started working on a fix for your processor description issue.  It was supposed to work, but the code that handles yaml was slightly wrong.

Comment (mathieu-artic) [reply tests/data/oneaccess_lbb350.json:2397]: i'm ready if you need some tests

**Review by murrant — APPROVED** [review]

**Pull request merged**